.github/workflows/publish-packages.yml

```diff
@@ -20,30 +20,31 @@ on:
 
 env:
   EVENT_NAME: ${{ github.event_name }}
+  PUBLISH: "true"
+  ARTIFACTORY_SECRET: ${{ secrets.ARTIFACTORY_SECRET }}
+  DEVOPS_PRIVATE_KEY: ${{ secrets.DEVOPS_PRIVATE_KEY }}
+  BINTRAY_PASSPHRASE: ${{ secrets.BINTRAY_PASSPHRASE }}
+  HZ_LICENSEKEY: ${{ secrets.HZ_LICENSEKEY }}
 
 # Constant for now - should ensure single build, maybe we can limit this to something from github.*
 concurrency: single-build
 
 jobs:
-  build:
+  prepare:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        java: [ '8' ]
-        architecture: [ 'x64' ]
         distribution: ['hazelcast', 'hazelcast-enterprise']
     env:
       HZ_VERSION: ${{ github.event.inputs.HZ_VERSION }}
       HZ_DISTRIBUTION: ${{ matrix.distribution }}
-      PUBLISH: "true"
-      ARTIFACTORY_SECRET: ${{ secrets.ARTIFACTORY_SECRET }}
-      DEVOPS_PRIVATE_KEY: ${{ secrets.DEVOPS_PRIVATE_KEY }}
-      BINTRAY_PASSPHRASE: ${{ secrets.BINTRAY_PASSPHRASE }}
     defaults:
       run:
         working-directory: ./hazelcast-packaging
     outputs:
+      # Beware that these should not be matrix specific - last matrix combination to finish overwrites previous
+      # executions
       hz_version: ${{ steps.hz_version.outputs.hz_version }}
       package_version: ${{ steps.package_version.outputs.package_version }}
     name: Hazelcast Packages Release Automation
```

Comment:
Could we use a different `name` or something shorter to see value from the matrix here:
![image](https://user-images.githubusercontent.com/1242724/153872571-802375a0-d99b-4922-ac4c-ca2810e1119d.png)

or maybe simply remove the `name` and stick with the job name?